### ISSUE TITLE
performance gap on 1080ti

### ISSUE THREAD

**Issue by ztt0821** [PJLallen/OSFormer#8]
Hi,  these days, I am running the programs on two 1080ti gpus. I keep all the default seeting, but the performance is too low. Here is my log file. I have checked the parameters, may I ask the random seed you used when you train the model. 
[log.txt](https://github.com/PJLallen/OSFormer/files/10048952/log.txt)

**Comment by Patrickctyyx** [comment]
Hi ztt, I noticed that you use CIS_RT(550) with `ENC_LAYERS = 3`, whose results should be lower than the reported results. Maybe you should set  `ENC_LAYERS = 6` and see if the result is still too low.
For the random seed, we did not set a seed manually.

**Comment by ztt0821** [comment]
Hi, I use the "CIS_RT.yaml" in the configs. The AP performance should be 36.0 on COD and 41.4 on NC4K. But my AP is 30.5 and 37.3. Maybe I need to test other lr. Now the program is running. I will tell you the result when I got. Thank you

**Comment by Patrickctyyx** [comment]
Hi, sorry for my misunderstanding. Did you use the pretrained weights from `detectron2://ImageNetPretrained/MSRA/R-50.pkl`? Since the format is different, weights from `detectron2://ImageNetPretrained/torchvision/R-50.pkl` shuold be lower.

**Comment by ztt0821** [comment]
Thank you for your information. I will test again and tell you the result tomorrow.

**Comment by ztt0821** [comment]
Hi, I have tested the CIS_RT.yaml on one 1080ti card, the performance is still lower than that on 3090 GPU card. I have use the pretrained weights from detectron2://ImageNetPretrained/MSRA/R-50.pkl. So I think this is the problem with GPU card accuracy.

**Comment by Patrickctyyx** [comment]
Have you tried to inference with our provided weights? Are the results lower than the reported?

**Comment by ztt0821** [comment]
Hi, I have tested the provided weights. I think the problem is the "error in ms_deformable_im2col_cuda: invalid device function
" error during the training process. I have trained the code on 3090 GPU that have the similar result as your paper. Thank you for your help.

**Comment by Patrickctyyx** [comment]
Thanks for your attention.
